### ISSUE TITLE
Firefox for Android webcam aspect ratio problem

### ISSUE THREAD

**Issue by hex-m** [bigbluebutton/bigbluebutton#14476]
**Describe the bug**
Depending on the camera profile and the orientation of the device, the the webcam image is very wide/high.

**To Reproduce**
Share a webcam in Firefox for Android. It looks okay when the quality is low. If you use the profile "high" or "high definition" the aspect ratio is wrong.

Reproducible on `test24.bigbluebutton.org`.

**Expected behavior**
The webcam should probably be either `4x3` or `16x9`? Not sure if it should be dependent on the orientation of the device.

**Screenshots**

![Screenshot from 2022-02-28 16-41-27](https://user-images.githubusercontent.com/16013327/156012503-4b39de7a-dfc2-4d3e-9fd2-217ab2ea7e0f.png)
![Screenshot from 2022-02-28 16-41-09](https://user-images.githubusercontent.com/16013327/156012509-85c22ec2-1867-4244-aa99-93e5917a9c2a.png)

**Smartphone (please complete the following information):**
 - Device: Fairphone 3 (FP3)
 - OS: Android 10
 - Browser: Firefox for Android, Fennec 97.1.1 (latest from f-droid)

**Additional context**
I can't explain why it only happens in high quality camera profiles. On my `2.4.4`-server it works fine with "Low" but breaks with "High".
```
      - id: low
        name: Low
        default: true
        bitrate: 100
      - id: high
        name: High
        default: false
        bitrate: 800
        constraints:
          width: 1280
          frameRate: 15
```

**Comment by prlanzarin** [comment]
I'm pretty sure this is a browser thing. Streams are displayed on video preview _as they're provided by the browser_.
If anything, maybe constraints could be tweaked, but it's still the browser that provides the capture stream that way.

Two suggestions:
  - Add the `height: 720` attribute to your high profile; restart bbb-html5, try again.
  - See whether it's reproducible by choosing `HD` here https://webrtc.github.io/samples/src/content/getusermedia/resolution/ (although this may fail with an overconstrained error due to the samples site using `exact` specifiers...)

**Comment by hex-m** [comment]
Thanks for your comment and help!

* Adding `height` helps. The aspect ration seems to be roughly `16x9` (or `9x16`) with this setting.
* Not reproducible. It works without error and the actual video dimensions are `720x1280px` and `1280x720px` (depending on the orientation).

**Comment by prlanzarin** [comment]
> Thanks for your comment and help!
> 
>     * Adding `height` helps. The aspect ration seems to be roughly `16x9` (or `9x16`) with this setting.
> 
>     * Not reproducible. It works without error and the actual video dimensions are `720x1280px` and `1280x720px` (depending on the orientation).

Thanks. I think I'll just add the `height` specifier set to 720 to the default high profile, then. Should mitigate this for the majority of scenarios.

I'll target v2.5, though, since it can be fixed via config in v2.4 and I'd like to minimize disruptions caused by potential overconstraining issues stemming from this change. It shouldn't cause problems, but I don't like altering constraints :).

**Comment by prlanzarin** [comment]
Addressed in #14513 / 2.5; as mentioned there, once I'm sure it doesn't bring overconstraining regressions of any sort I'll backport it to 2.4.

**Comment by prlanzarin** [comment]
2.4 backport https://github.com/bigbluebutton/bigbluebutton/pull/14592
